src/components/side-panel/common/dataset-tag.tsx

```diff
@@ -48,12 +49,12 @@ const DatasetColorPicker = styled.div`
   display: flex;
 `;
 
-const UpdateTableColor = ({updateTableColor = nop, children, id}) => (
+const UpdateTableColor = ({updateTableColor = nop, children, id}: UpdateTableColorTypes) => (
   <DatasetColorPicker
     className="dataset-action update-color"
     data-tip
     data-for={`update-color-${id}`}
-    onClick={e => {
+    onClick={(e: MouseEvent) => {
       e.stopPropagation();
       updateTableColor(id);
```

Comment:
@heshan0131 here is place where updateTableColor called with no color parameter. What I should place here?

Reply:
`updateTableColor` is not used in the component, the `onClick` prop should be removed from `DatasetColorPicker`

Also remove `updateTableColor` prop from `DatasetTagWrapper` and `UpdateTableColor`

```
-    <DatasetTagWrapper className="source-data-tag" updateTableColor={updateTableColor}>
-      <UpdateTableColor id={dataset.id} updateTableColor={updateTableColor}>
+    <DatasetTagWrapper className="source-data-tag">
+      <UpdateTableColor id={dataset.id}>

```

Reply:
fixed